### PR TITLE
Testing login use case

### PR DESCRIPTION
**Pull request by Shaffaan**
Created 3 tests for "Create User and fridge test" and 3 tests for "Login Use Case". Multiple test cases are implemented to make sure program runs according to specification

### CODE REVIEW

File: src/test/java/test_login_use_case/TestLoginUseCase.java [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+    /**
+     * Expected Result when nothing is entered
+     */
+    @Test
+    void nothingTest(){
+        DatabaseGateway testingDatabase = new Database("Test_Storage");
+        loginPresenter testingPresenter = new loginPresenter() {
+            @Override
+            public loginResponseModel prepareSuccessView(loginResponseModel responseModel) {
+                return null;
```
Comment (rickychen175) [inline src/test/java/test_login_use_case/TestLoginUseCase.java:104]: I think you should add an assertion statement that fails here. If a future change accidentally makes login succeed for no username entered, the prepareSuccessView() function would run and the test would pass. The tests for Registration may be affected too.

File: src/test/java/test_login_use_case/TestLoginUseCase.java [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+            @Override
+            public loginResponseModel prepareFailView(String error) {
+                assertEquals(error, "Account does not exist");
+                return null;
+            }
+        };
+        loginInputBoundary testing_login = new LoginInteractor(testingDatabase, testingPresenter);
+        loginRequestModel inputData = new loginRequestModel("demetriusSr");
+        testing_login.create(inputData);
+
```
Comment (rickychen175) [inline src/test/java/test_login_use_case/TestLoginUseCase.java:91]: I would suggest deleting the database storage file after every test. This avoids tests possibly interfering with other tests if we create more later.

Comment (Shaffaan) [reply src/test/java/test_login_use_case/TestLoginUseCase.java:91]: yeah for sure, I think I forgot to add it here

File: src/test/java/test_login_use_case/TestLoginUseCase.java (hunk 1 above)
Comment (Shaffaan) [reply src/test/java/test_login_use_case/TestLoginUseCase.java:104]: I am not quite sure what the assertion statement would check, do you mind elaborating.

File: src/main/java/use_cases/create_user_and_fridge/CreateUserandFridgeInteractor.java [hunk 3]
```diff
@@ -48,11 +48,11 @@ public CreateUserandFridgeResponseModel create(CreateUserandFridgeRequestModel r
 /**
  * Need some method here that already checks if database has it
  */
```
Comment (fishyvishy) [inline src/main/java/use_cases/create_user_and_fridge/CreateUserandFridgeInteractor.java]: should this comment be deleted?

File: src/test/java/test_login_use_case/TestLoginUseCase.java (hunk 2 above)
Comment (EricKarpovits) [reply src/test/java/test_login_use_case/TestLoginUseCase.java:91]: You can look at how I did it.

**Review by EricKarpovits — APPROVED** [review]

**Review by fishyvishy — APPROVED** [review]
Well done. Tests seem to cover all edge cases

**Review by rickychen175 — APPROVED** [review]
